src/math_defines.h

```diff
@@ -13,7 +13,7 @@
 // https://stackoverflow.com/questions/6563810/m-pi-works-with-math-h-but-not-with-cmath-in-visual-studio/6563891
 
 #define _USE_MATH_DEFINES
-#include <cmath>
+#include <math.h>
```

Comment:
I think you'll need a clang-tidy suppression here.

Reply:
That's true.